server/compute.js

```diff
@@ -20,6 +22,46 @@ function applyUtils(sandbox = {}) {
   sandbox._ = deepFreeze(lodash);
 }
 
+const buildPayload = (pld, pl = {}) => {
```

Comment:
We want to call `client.asUser().traits` only once for optimization reasons. And this way we can compute the changes made, and call `client.asUser().traits` only if needed. Also we want to display the changes in the preview.